Zend/zend_interfaces.c

```diff
@@ -33,6 +33,20 @@ ZEND_API zend_class_entry *zend_ce_stringable;
 /* {{{ zend_call_method
  Only returns the returned zval if retval_ptr != NULL */
 ZEND_API zval* zend_call_method(zend_object *object, zend_class_entry *obj_ce, zend_function **fn_proxy, const char *function_name, size_t function_name_len, zval *retval_ptr, int param_count, zval* arg1, zval* arg2)
+{
+	zend_string *function_name_str;
+	zval *retval;
+
+	function_name_str = zend_string_init(function_name, function_name_len, 0);
```

Comment:
This looks like a regression to me. Previously, the function name string was only created if the `fn_proxy` cache has not been populated yet. Now it will always be created instead.

Reply:
> This looks like a regression to me. Previously, the function name string was only created if the `fn_proxy` cache has not been populated yet. Now it will always be created instead.

it looks better now